metrics/default-mobile-app-metrics.md

```diff
@@ -0,0 +1,139 @@
+# Abstract
+
+This document contains the proposal for gathering some default metrics of mobile apps and making them available to developers.
+
+## Goals
+
+### Goals
+
+- Provide mobile app developers a service that can receive some basic default metrics, namely sdk version and app version
+
+- Provide developers an interface to query and visualize the gathered data in order to make informed decisions
+
+### Non Goals
+
+- Identify individual devices or users in metrics events (bonus)
+
+- Registering of user-defined metrics and logs (not prioritized; just keep in mind)
+
+- Provide mobile app developers with an easy to use SDK to connect easily and securely to the metrics service (separate epic/proposal)
+
+## Problem Description
+
+As mobile apps are developed, improved and iterated on, different versions of those apps can be used on many different devices. 
+Metrics data becomes invaluable for developers to observe the landscape of application installations and make sure further 
+development and deployment of backend services will not affect users.
+
+## Proposed Solution
+
+A Metrics Service will be available from the Service Catalog, which can be provisioned by developers for use with their mobile applications.
+
+The following diagram illustrates the components:
+
+![service diagram](./default-mobile-metrics-system-landscape.png)
+
+### Mobile App Client
+
+A native, platform-specific SDK for targeting the Metrics Service will be available to developers.
+
+Upon installation into a project and providing configuration for targeting the Metrics Service the SDK will automatically 
+send the default metrics information for the application with a unique ID to the metrics service.
+This information will be sent to the service every time the application starts.  
+
+Please note that, the client side is not part of this proposal.
```

Comment:
Let's remove that statement. Mobile metrics are for mobile apps. Considering that mobile part is mostly done I think we should just explain interface for mobile. There is no need for separate proposal for mobile.